data/cybox/nozomi/nozomi_alerts_22022024.json

```diff
@@ -0,0 +1,563 @@
+                        "mitre-attack-ext": {
+                            "technique_id": "T1095",
+                            "technique_name": "Non-Application Layer Protocol",
+                            "tactic_name": "Command and Control"
+                        }
+                    },
+                    "name": "Non-Application Layer Protocol",
+                    "kill_chain_phases": [
+                        {
+                            "kill_chain_name": "mitre-attack",
+                            "phase_name": "Command and Control"
+                        }
+                    ]
+                }
+            },
+            "first_observed": "2023-12-14T09:14:39.802Z",
+            "last_observed": "2023-12-14T09:14:39.802Z",
+            "number_observed": 1
+        },
+		{
+            "id": "observed-data--d7670487-3811-4e16-95f5-333f82b94a74",
+            "type": "observed-data",
+            "created_by_ref": "identity--f431f809-377b-45e0-aa1c-6a4751cae5ff",
+            "created": "2024-02-22T06:51:20.243Z",
+            "modified": "2024-02-22T06:51:20.243Z",
+            "objects": {
+                "0": {
+                    "type": "x-ibm-finding",
+                    "alert_id": "201445c6-a120-5ab9-9703-cb2119d55f23",
+                    "x_is_acknowledged": false,
+                    "severity": 93,
+                    "time_observed": "2024-01-09T09:19:18.000Z",
+                    "dst_ip_ref": "1",
+                    "src_ip_ref": "3",
+                    "x_alert_status": "open",
+                    "name": "Packet rule match",
+                    "x_cause": "Activity was detected related to an exploit at the SMB protocol - Eternal Blue. The SMBv1 server in Microsoft Windows Vista SP2, Windows Server 2008 SP2 and R2 SP1, Windows 7 SP1, Windows 8.1, Windows Server 2012 Gold and R2, Windows RT 8.1, and Windows 10 Gold, 1511, and 1607, and Windows Server 2016 allows remote attackers to execute arbitrary code via crafted packets.",
+                    "x_message": "Microsoft Windows SMB remote code execution attempt.",
+                    "ioc_refs": [
+                        "8"
+                    ],
+                    "x_solution": "Investigate the involved Assets and IPs to check their status and ensure that the system is not compromised.",
+                    "x_cve_references": [
+                        "2017-0144",
+                        "2017-0146"
+                    ],
+                    "description": "A suspicious packet was sent [sid:41978] -- Microsoft Windows SMB remote code execution attempt. Activity was detected related to an exploit at the SMB protocol - Eternal Blue. The SMBv1 server in Microsoft Windows Vista SP2, Windows Server 2008 SP2 and R2 SP1, Windows 7 SP1, Windows 8.1, Windows Server 2012 Gold and R2, Windows RT 8.1, and Windows 10 Gold, 1511, and 1607, and Windows Server 2016 allows remote attackers to execute arbitrary code via crafted packets.",
```

Comment:
sid's may be considered PI, can you replace this?

Reply:
Please use a valid but clearly false MAC address, such as "01:01:01:01:01:01"

Reply:
Updated as per the suggestion.